sql_generators/funnels/configs/iOS_onboarding.toml

```diff
@@ -0,0 +1,368 @@
+destination_dataset = "firefox_ios_derived"
+platform = "firefox_ios"
+owners = ["rzhao@mozilla.org"]  # optional; users getting notification if funnel run fails
+version = "1"  # optional; default is set to 1
+
+[funnels]
+
+[funnels.ios_onboarding_funnel]
+
+friendly_name = "Firefox for iOS Onboarding Funnel"
+description = "Funnel Steps for Firefox for iOS Onboarding"
+steps = [
+        "new_profile",
+        "first_card_impression",
+        "first_card_primary_click",
+        "first_card_secondary_click",
+        "first_card_close_click",
+        "second_card_impression",
+        "second_card_primary_click",
+        "second_card_secondary_click",
+        "second_card_close_click",
+        "third_card_impression",
+        "third_card_primary_click",
+        "third_card_secondary_click",
+        "third_card_close_click",
+        "fourth_card_impression",
+        "fourth_card_primary_click",
+        "fourth_card_secondary_click",
+        "fourth_card_close_click",
+        "fifth_card_impression",
+        "fifth_card_primary_click",
+        "fifth_card_secondary_click",
+        "fifth_card_close_click",
+         "sync_sign_in",
+         ]
+
+
```

Comment:
Do we need `[steps]` here?

For example I can see in another similar config this section being defined:
https://github.com/mozilla/bigquery-etl/blob/ece677485b8c30c0bbc92552851ed68fbabe2840/sql_generators/funnels/configs/email_first_reg_login_funnels_by_service.toml#L22